controllers/databasecluster_controller.go

```diff
@@ -3101,6 +3121,10 @@ func getPGRetentionCopies(retentionCopies int32) int {
 	return copies
 }
 
+func backupStoragePrefix(db *everestv1alpha1.DatabaseCluster) string {
+	return fmt.Sprintf("%s/%s", db.Name, db.UID)
+}
```

Comment:
the case when we have two BS referencing the same bucket is still possible, so do we need the storage name there?

Reply:
I don't think we do.
Backup storage defines a way to access a certain storage. In case two backup storages point to the same location, it's similar to having two different sets of credentials.

Reply:
Are you suggesting we shall have a check in place to make sure no two backup storages refer to the same location (e.g. the same bucket)?

Reply:
That could be an option. Another option is have the storage name in the prefix. 
Both are handling the case with the same bucket so it won't break the functionality.